packages/@okta/vuepress-site/docs/guides/third-party-risk-integration/test-integration/index.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Test the integration
+---
+
+With the creation of a service application for the third-party risk provider, and the update of the third-party risk provider profile, you can now test the integration using the Risk Events API. In this test, the API sends a sample payload risk event to the Okta org, which can be consumed by Okta and used to calculate the risk of the authentication.
```

Comment:
I think it may be helpful if we indicate here that the `risk-events` API is something the partner hits to send Okta the risk events. The partner would need the public/private key-pair and the clientId from the previous setup in order to do that. In other words, the other parts of the setup (setting up the service app, updating the risk provider configuration, etc. ) is something the admin is supposed to do and the parther is supposed to generate the client assertion and use that to generate a JWT to make the risk-events API call.

Reply:
Thanks, Raghu. Added this information to this location. I didn't use the term "partner" as I did not use it elsewhere in the guide, and referred only to the "third-party risk provider".